src/components/Content/Page.js

```diff
@@ -12,6 +12,8 @@ class Page extends React.Component {
     /* We can insert our html here directly since we trust our backend cms */
     return (
       <div>
+        <div className={style.remoteHeading}
+             dangerouslySetInnerHTML={{__html: (this.props.page.title)}}/>
```

Comment:
Can't we adjust this to `<div className={style.remoteHeading}>{ this.props.page.title}</div>`?

Reply:
Probably titles are html, but I could be wrong :P

Reply:
Or... I thtink they shouldn't. hmmm

Reply:
done